### ISSUE TITLE
uninitialized constant JiraScan::StringIO

### ISSUE THREAD

**Issue by cosad3s** [bcoles/jira_scan#2]
I try to use jira_scan against JIRA instance and I got an error:

```bash
I, [2023-01-12 18:16:26  #18]  INFO -- jira-scan: Fetching https://xxxxxxxxxxxxx/secure/Dashboard.jspa
E, [2023-01-12 18:16:26  #18] ERROR -- jira-scan: Could not retrieve URL https://xxxxxxxxxxxxx/secure/Dashboard.jspa: uninitialized constant JiraScan::StringIO
```

I have tested jira_scan with Ruby 3.2.0.
Same error in Dockerized Ruby.

**Comment by bcoles** [comment]
Does the following diff fix the issue for you?

```diff
diff --git a/lib/jira_scan.rb b/lib/jira_scan.rb
index 6687e0e..234c911 100644
--- a/lib/jira_scan.rb
+++ b/lib/jira_scan.rb
@@ -9,6 +9,7 @@ require 'json'
 require 'logger'
 require 'net/http'
 require 'openssl'
+require 'stringio'
 
 class JiraScan
   VERSION = '0.0.5'.freeze
```

I can't reproduce this on my test system with Ruby 3.1.2.

**Comment by cosad3s** [comment]
To reproduce in Docker: `docker run -it ruby bash`, then install / build / run the scanner as usual.

I have tested your fix: that works :) Thank you !

The same problem occurs for your other scanner for Sitecore: https://github.com/bcoles/sitecore_scan

**Comment by bcoles** [comment]
> To reproduce in Docker: `docker run -it ruby bash`, then install / build / run the scanner as usual.
> 
> I have tested your fix: that works :) Thank you !
> 
> The same problem occurs for your other scanner for Sitecore: https://github.com/bcoles/sitecore_scan

Thanks. Yeah it probably affects all of my `Scan*` tools.

**Comment by bcoles** [comment]
This should be fixed in version 0.0.6.
